src/TabComplete.js

```diff
@@ -307,6 +326,49 @@ class TabComplete {
             this.opts.onStateChange(this.completing);
         }
     }
+
+    _sortEntries() {
+        // largest comes first
+        const KIND_ORDER = {
+            command: 1,
+            member: 2,
+        };
+
+        this.list.sort((a, b) => {
```

Comment:
I don't believe it's specified, although you're correct: I'm assuming that sorting a very-nearly sorted list should be fast: at worst, I think it's about the same which, for the number of things we're sorting, is probably fine.